docs/developer-tools/runtime/services/marlowe-indexer.md

```diff
@@ -5,18 +5,31 @@ title: Marlowe indexer service
 The `marlowe-indexer` executable follows `marlowe-chain-sync` and writes Marlowe contract transactions to a database.
 
 ```console
-marlowe-indexer : a contract indexing service for the Marlowe Runtime.
+marlowe-indexer: Contract indexing service for the Marlowe Runtime.
 
-Usage: marlowe-indexer [--chain-sync-port PORT_NUMBER]
-                       [--chain-sync-query-port PORT_NUMBER]
+Usage: marlowe-indexer [--version] [--chain-sync-port PORT_NUMBER] 
+                       [--chain-sync-query-port PORT_NUMBER] 
                        [--chain-sync-host HOST_NAME]
-                       (-d|--database-uri DATABASE_URI)
+                       (-d|--database-uri DATABASE_URI) 
                        [--http-port PORT_NUMBER]
 
-  Contract indexing service for Marlowe Runtime
+  The contract indexer for the Marlowe Runtime. It connects to a marlowe-chain-sync
+  instance using both the chain seek and chain query protocol. It scans the chain for
+  Marlowe contract transactions and saves them in a postgresql database. This database
+  can be queried by downstream components, such as marlowe-sync.
+
+  There should only be one instance of marlowe-indexer writing data to a given marlowe
```

Comment:
Suggestion: 

"...to a given Marlowe database."

Reply:
```suggestion
  There should only be one instance of marlowe-indexer writing data to a given Marlowe
```